api/server/router/system/system_routes.go

```diff
@@ -89,51 +100,89 @@ func (s *systemRouter) getVersion(ctx context.Context, w http.ResponseWriter, r
 	return httputils.WriteJSON(w, http.StatusOK, info)
 }
 
+// DiskUsageObject represents an object type used for disk usage query filtering.
+type DiskUsageObject string
+
+const (
+	// ContainerObject represents a container DiskUsageObject.
+	ContainerObject DiskUsageObject = "container"
```

Comment:
nit: one of the example you provided used `containers`, `images`, `volumes`. I personnaly prefer with the s

Reply:
https://github.com/moby/moby/pull/42559/commits/72be7d4c6ada92271eb8a6f5f562427efa815ab6